modules/graphstudio/pages/explore-graph/actions.adoc

```diff
@@ -0,0 +1,38 @@
+= Actions
+
+The Actions tab in GraphStudio is a hub for actions that affect the entire graph database.
+
+== Clear all graph data
+
+image::clear-all-graph-data.png[]
+
+This command removes all loaded data from the schema while retaining all graph schemas and queries.
+
+Your user account must have the superuser role, and you must also be in Global View to clear all data from all graphs.
```

Comment:
```suggestion
Your user account must have the superuser role or have a role with the clear graph store privilege, and you must also be in Global View to clear all data from all graphs.
```

Reply:
Good callout, we should try to change instances of superuser whenever we see them in the documentation .